CMakeLists.txt

```diff
@@ -2,19 +2,14 @@ cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 project(csn)
 
 option(BUILD_PATCHDD "build patchdd utility" OFF)
+option(BUILD_CSN "build main casync tool" ON)
+option(BUILD_TESTING "build casync tool tests" ON)
```

Comment:
`include(CTest)` will create this option. Is there any reason why you had to pull this out here (and below)?

Reply:
I didn't know this, I thought this was implicitly created somewhere :+1: